db/migrate/20221020182834_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :post, null: false, foreign_key: true
```

Comment:
- [ ] Kindly also here write the query like this so the foreign key name will be `author_id`.

```ruby
t.references :author, foreign_key: { to_table: 'users' }
```